src/Library/demos/List Model/main.blp

```diff
@@ -0,0 +1,94 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "List Model";
+  description: _("List models are a simple interface for ordered lists of GObject instances");
+
+  Box {
+    orientation: vertical;
+    spacing: 24;
+
+    Gtk.StackSwitcher {
+      stack: stack;
+      halign: center;
+    }
+
+    Gtk.Stack stack {
+      transition-duration: 300;
+      transition-type: crossfade;
+      vexpand: true;
+
+      Gtk.StackPage {
+        name: "listbox";
+        title: _("List Box");
+        child:
+          Box {
+            halign: center;
+            ListBox list_box {
+              selection-mode: none;
+              width-request: 240;
+              styles ["boxed-list"]
+            }
+          };
+      }
+
+      Gtk.StackPage {
+        name: "flowbox";
+        title: _("Flow Box");
+        child:
+          Box {
+            halign: center;
+            FlowBox flow_box {
+              width-request: 240;
+              orientation: horizontal;
+              selection-mode: none;
+              styles ["card"]
+            }
+          };
+      }
+
+
+      Gtk.StackPage {
+        name: "Filter";
+        title: _("Edit");
+        child:
+          Box {
+            orientation: vertical;
+            spacing: 12;
+            halign: center;
+
+            SearchEntry search_entry {
+              halign: center;
+              placeholder-text: _("Start searching");
+            }
+
+            ListBox list_box_editable {
+              width-request: 240;
+              activate-on-single-click: true;
+              selection-mode: none;
```

Comment:
Let's set this to `single` and mark the remove button `active` when there is a selected row, then remove the item from the model with at the row's index position. I think that's sensible UX.